packages/calcite-design-tokens/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to calcite-design-tokens
+
+You want to contribute? Nice! Below are some guidelines for ensuring that your contribution makes sense for everybody.
+
+## Reporting Issues
+
+Found a problem? Want a new feature?
+
+- See if your issue or idea has [already been reported](issues).
+- Provide detailed reproduction instructions as well as what behavior is expected.
+
+## Contributing to tokens
+
+### Read [<root>/CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+These docs include initial setup, commit message format guidelines and more.
+
+### Project scripts
+
+#### Build
+
+Build the tokens to platform assets
+
+`npm --workspace=packages/calcite-design-tokens run build`
+
+## Understanding Token Files
+
+All token files may be updated by the Figma Token Studio plugin. These can also be updated in the code directly. Any changes to token files must be reviewed by the Calcite Design Team.
+
+| Name                 | Description                                                                                                       |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `src/$metadata.json` | This file is used by Figma Token Studio but is not used to determine final token asset output.                    |
+| `src/$themes.json`   | Each theme defined in this file will output a token asset in each format defined in the token transformer config. |
+| `src/core.json`      | The core design tokens upon which the rest of the tokens are built.                                               |
+| `src/semantic.json`  | The semantic design tokens used by theme files and components.                                                    |
+| `src/component/`     | The component design tokens. These are used by themes.                                                            |
+| `src/calcite/`       | The Calcite themes. This currently consists of a light and dark theme.                                            |
+
+## Submitting Pull Requests
```

Comment:
This section can be removed since we discuss submitting PRs in the general contributing doc. If you want to touch on `npm test`, can you move it to the `Project scripts` section above?